__tests__/packageRepoUtils.test.js

```diff
@@ -54,47 +53,35 @@ test('repo utils retrieves package latest version', async () => {
   expect(result).toEqual('3.1.0')
 })
 
+test('repo utils retrieves package README information', async () => {
+  const packageRepoUtils = new PackageRepoUtils()
+  const packageName = 'testPackage'
+  const result = await packageRepoUtils.getReadmeInfo(packageName)
+  expect(result).toContain('dockly')
+})
+
 test('repo utils retrieves package latest version as null if not exists', async () => {
   const PackageRepoUtils = require('../lib/helpers/packageRepoUtils')
-  jest.mock('axios', () => {
-    return {
-      get: jest.fn(() => {
-        return Promise.resolve({
-          data: require('./mocks/registryPackageUnpublished.mock.json')
-        })
-      })
-    }
-  })
+  fetch.mockReturnValue(Promise.resolve({
+    data: require('./mocks/registryPackageUnpublished.mock.json')
+  }))
```

Comment:
when I write 
```
jest.mock('node-fetch', () => {
    return {
      get: jest.fn(() => {
        return Promise.resolve({
          data: require('./mocks/registryPackageUnpublished.mock.json')
        })
      })
    }
  })
```
, test failed.

<img width="879" alt="スクリーンショット 2020-11-26 10 07 45" src="https://user-images.githubusercontent.com/50798936/100296327-40648f80-2fcf-11eb-8385-218ccbe00f80.png">

Other tests below failed as well when write `jest.mock('node-fetch',...` .

Reply:
Shouldn't it rather be:
```javascript
jest.mock('fetch', () => {
  return jest.fn(() => {
    return Promise.resolve({
      data: require('./mocks/registryPackageUnpublished.mock.json')
    })
  })
})
```
Since you use `fetch` directly with `fetch()` rather than axios' submethod `axios.get()`

Reply:
thanks @xiniria